sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_link_async.py

```diff
@@ -182,11 +183,12 @@ async def _incoming_attach(self, frame):
         if self.network_trace:
             _LOGGER.info("<- %r", AttachFrame(*frame), extra=self.network_trace_params)
         if self._is_closed:
-            raise ValueError("Invalid link")
+            raise AMQPLinkError(condition=ErrorCondition.ClientError, description="Invalid link", info=None)
```

Comment:
Could we update the `description` to something like "Received Attach frame on a link that is already closed."

Reply:
Same comment applies to the sync `link.py`